nomenclature/definitions/variable/emissions/emissions.yaml

```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
                 (IPCC category 3)
    unit: Mt CO2/yr
 
+Emmissions|CO2|Electricity:
+   description: CO2 emmissions of power plants .
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Biomass|Traditionnal:
+   description: CO2 emmissions of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Biomass|New and imported :
+   description: CO2 emmissions of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Coal:
+   description: CO2 emmissions of a typical coal power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Gas|OCGT:
```

Comment:
```suggestion
Emission Rate|CO2|Electricity|Gas|OCGT:
```

Reply:
I made it locally some days ago because the commit suggestion was outdated